.github/workflows/main.yml

```diff
@@ -38,11 +38,11 @@ jobs:
               os: [self-hosted, Linux, skylake-2x],
             }
           - { rust: nightly, target: x86_64-apple-darwin, os: macos-latest }
-          - {
-              rust: nightly,
-              target: x86_64-pc-windows-msvc,
-              os: windows-latest,
-            }
+          #- {
```

Comment:
Giving up on windows completely is a bit too radical.  Perhaps we could just build/test with minimal features?

Reply:
It runs out of disk-space during build -- I'll try to build with less features/dependencies.

Reply:
Even running with no features would be fairly useful, compared to dropping windows altogether.

Reply:
Disabling the CI cache for windows makes it work again with no need to limit features.